src/GapSkipper.js

```diff
@@ -0,0 +1,85 @@
+import videojs from 'video.js';
```

Comment:
I would rename this file gap-skipper to be more in line with the other files in the src folder.